src/Consolonia.Themes/Templates/Controls/Button.axaml

```diff
@@ -126,12 +126,12 @@
 
         <Style Selector="^ /template/ Border#InternalBorder">
             <Setter Property="Margin"
-                    Value="0,0,1,1" />
+                    Value="0,0,1,0" />
```

Comment:
Looks like turboVision theme is broken with this changes

Reply:
Oh shoot, I forgot about that. 

Yeah, the issue is that this makes all buttons 2 characters high all of the time.  This was making the title bar 2 characters high.  

What do you want Turbo Vision theme to look like for dialog titles?

Reply:
I fixed it. I updated the button styles in turbo to be approraite thing